RevenueCat/Scripts/PurchasesConfiguration.cs

```diff
@@ -41,6 +52,7 @@ public static Builder Init(string apiKey)
 
             public PurchasesConfiguration Build()
             {
+                _dangerousSettings ??= new DangerousSettings(false);
```

Comment:
without this, creating a build but not setting dangerous settings would crash

Reply:
I guess we don't have tests for this right?

Reply:
we don't have unit tests for anything in Unity, sadly. Hoping to get that started soon. We do have API tests, as well as integration tests that export a `.unityPackage`, import it into an iOS and an Android app, and try building (and creating an archive on iOS)